deployer/cli.py

```diff
@@ -64,15 +64,18 @@ def main():
         nargs="?",
         help="The hub, or list of hubs, to install/upgrade the helm chart for",
     )
-    deploy_parser.add_argument(
-        "--skip-hub-health-test", action="store_true", help="Bypass the hub health test"
-    )
     deploy_parser.add_argument(
         "--config-path",
         help="File to read secret deployment configuration from",
         # This filepath is relative to the PROJECT ROOT
         default="shared/deployer/enc-auth-providers-credentials.secret.yaml",
     )
+    deploy_parser.add_argument(
```

Comment:
Can we make a note in daskhub/Chart.yaml where we specify the daskhub version that if that is bumped, this default also needs to be changed?